src/main/java/org/jboss/pnc/deliverablesanalyzer/model/FinderResultCreator.java

```diff
@@ -60,6 +64,50 @@ public static FinderResult createFinderResult(String id, URL url, Map<BuildSyste
                 .build();
     }
 
+    private static void setLicenseInformation(Artifact.ArtifactBuilder builder, KojiLocalArchive localArchive) {
+        Set<LicenseInfo> licenses = Optional.ofNullable(localArchive.getLicenses())
+                .orElse(Collections.emptySet())
+                .stream()
+                .map((org.jboss.pnc.build.finder.core.LicenseInfo license) -> {
```

Comment:
I would prefer the labmda to be a method (e.g. `private static LicenseInfo toDTO(org.jboss.pnc.build.finder.core.LicenseInfo)`) and have it referenced in the map (`map(FinderResultCreator::toDTO)`). I think it makes it more readable.